java/Cherepnin_Oleh/src/com/zoolatech/lecture5/tasks/_3/TaskThree.java

```diff
@@ -0,0 +1,54 @@
+package com.zoolatech.lecture5.tasks._3;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Create a method that accepts a list of words, and displays only unique one (order can be random).
+ * Bonus task: treat strings in a different case as the same one: “Apple” and “apple” should be treated
+ * as the same string (don’t use any methods that create a new string object - like toUpperCase or
+ * toLowerCase - you can work only with a given input).
+ * Input: [“apple”, “orange”, “pear”, “apple”, “banana”, “orange”]
+ * Output: apple
+ * orange
+ * pear
+ * banana
+ */
+public class TaskThree {
+    public static void main(String[] args) {
+        String[] fruits = {"apple", "Apple", "orange", "orange", "pear", "peach", "Pear", "peach"};
+        displayUnique(fruits);
+        displayUniqueIgnoreCase(fruits);
+    }
+
+    static void displayUnique(String[] stringList) {
+        List<String> distinct = Arrays.stream(stringList)
+                .distinct()
+                .collect(Collectors.toList());
+        System.out.println(distinct);
+    }
+
+    static void displayUniqueIgnoreCase(String[] stringArray) {
```

Comment:
can you find a solution that takes only one loop to find a solution? hint: check sets